CHANGELOG_iOS.md

```diff
@@ -1,5 +1,33 @@
 # Changelog
 
+## [1.81.134](https://github.com/brave/brave-browser/releases/tag/v1.81.134)
+
+### Web3 
+
+ - Added support for resolving new Unstoppable Domains TLDs including .brave TLD. ([#46271](https://github.com/brave/brave-browser/issues/46271))
+ - Added a Brave proxy for Zcash mainnet endpoint. ([#46266](https://github.com/brave/brave-browser/issues/46266))
+ - Enabled ZCash feature flag by default. ([#48171](https://github.com/brave/brave-browser/issues/48171))
+ - Fixed issue with web-views not reloading when default Solana Wallet provider is changed. ([#47676](https://github.com/brave/brave-browser/issues/47676))
```

Comment:
nit: should change it to `web views` like we're doing via https://github.com/brave/brave-browser/pull/48324/files#diff-41c311cb76364cc951915ffbde6a0cb335a36c4766523edd90cedee18d3c81ecR27. Looks like Apple uses `web views` rather than `web-views` as per https://developer.apple.com/design/human-interface-guidelines/web-views.